pkg/volume/storageos/storageos_util.go

```diff
@@ -322,31 +322,28 @@ func getLoopDevice(path string) (string, error) {
 		return "", fmt.Errorf("not attachable: %v", err)
 	}
 
-	exec := exec.New()
 	args := []string{"-j", path}
-	out, err := exec.Command(losetupPath, args...).CombinedOutput()
+	out, err := exec.Run(losetupPath, args...)
```

Comment:
I need to test this to make sure we don't need .CombinedOutput() to correctly parse the info we need.